Nos/Service/Analytics.swift

```diff
@@ -116,7 +116,7 @@ class Analytics {
         track("Imported Private Key")
     }
     
-    func changedKey() {
+    func changedKey() { // TODO: delete? it's unused
```

Comment:
yeah this can go. Now you have to log out to do this.

Reply:
deleted